_pages/06-publications.md

```diff
@@ -203,7 +203,7 @@ The interest for Quality of Service (QoS) measurements in the Internet has excee
 {:.paper-author}
 Ioannis Koukoutsidis
 
-[Download PDF](http://www.jstor.org/stable/pdf/10.5325/jinfopoli.5.2015.0245.pdf){:.download-link .paper-download target="_blank"}
+Download link: http://www.jstor.org/stable/pdf/10.5325/jinfopoli.5.2015.0245.pdf
```

Comment:
Does this need to be linkified?

Reply:
I had to remove the markdown that makes this a link. JSTOR now doesn't let you link directly to a resource unless you're logged into JSTOR. If you're not logged in, it redirects you to a Terms and Conditions acceptance page first. So in the site build, the link checker can't resolve the link. See screenshot. My resolution was to just provide the URL. Open to suggestions.
![image](https://user-images.githubusercontent.com/4633345/48062825-26aa4280-e191-11e8-80cf-f2612a0b6596.png)